packages/cli-build/src/commands/build/finalize.js

```diff
@@ -26,7 +27,10 @@ export class Finalize extends Command {
       process.env.PERCY_PARALLEL_TOTAL = '-1';
     }
 
-    let client = new PercyClient();
+    let client = new PercyClient({
+      clientInfo: `${pkg.name}/${pkg.version}`,
+      environmentInfo: ''
```

Comment:
Now back to these. With those changes, we should be safe to repeat node info. It might help rule out any crazy environments if multiple node versions show up.